src/Command/Docker/Build.php

```diff
@@ -116,32 +116,32 @@ protected function configure()
             ->addOption(
                 self::OPTION_PHP,
                 null,
-                InputOption::VALUE_OPTIONAL,
+                InputOption::VALUE_REQUIRED,
                 'PHP version'
             )->addOption(
                 self::OPTION_NGINX,
                 null,
-                InputOption::VALUE_OPTIONAL,
+                InputOption::VALUE_REQUIRED,
```

Comment:
It must be optional because by default we're parsing `.magetno.env.yaml` file

Reply:
It's not about option it's about value.
With `InputOption::VALUE_OPTIONAL` you can pass `--nginx` without value.
With `InputOption::VALUE_REQUIRED` you can't pass `--nginx` without value.
An option will still be optional but can't be added without value.

Reply:
Got it, thanks for clarification